tasks-by-users/Naufil-Shemle/06-shoe-card-UI-design/style.css

```diff
@@ -0,0 +1,224 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600;700;800;900&display=swap");
+* {
+  margin: 0;
+  padding: 0;
+  font-family: "Poppins", sans-serif;
```

Comment:
Is it enough for css reset? @Naufil-Shemle

Reply:
`box-sizing` is missing, he didn't use in that tutorial so that I also didn't use